scripts/qtcreator/templates/wizards/openframeworks/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include "ofMain.h"
```

Comment:
yes i know but the wizard needs to have the template in it's own folder to be able to find it once it's copied to the qtcreator install. 

i plan to extend this further being able to specify the OF path in a config option in qtcreator... and then it'll be possible to get the template from the standard location, use custom templates... but by now we need this here